core/events.go

```diff
@@ -30,6 +30,11 @@ type TxPreEvent struct{ Tx *types.Transaction }
 // TxPostEvent is posted when a transaction has been processed.
 type TxPostEvent struct{ Tx *types.Transaction }
 
+// PendingLogsEvent is posted post mining and notifies of pending logs.
```

Comment:
Post or *pre* mining? Aren't these supposed to fire when the miner starts working on them?